x-pack/plugins/observability/public/rules/create_observability_rule_type_registry.ts

```diff
@@ -14,10 +14,12 @@ type Formatter = (options: {
   formatters: { asDuration: AsDuration; asPercent: AsPercent };
 }) => { reason: string; link: string };
 
+export type FormattableAlertTypeModel = AlertTypeModel<any> & { format: Formatter };
```

Comment:
can we replace this `any`?

Reply:
Don't want to change too much of the types in observability. I only pulled it out. @jasonrhodes any insight into how we can make this more strongly typed. I know Dario was likely the one who worked on this.

Reply:
Interestingly enough, this is actually the same type used trigger actions ui, so I think it's best to leave this to be determined by the alerting team or Dario.

Reply:
Hey, makes sense. For the time being, we kind of _are_ the alerting team (that manages this rule registry) and I'm working on liaising between everyone who is currently touching it. 

If any is being used already with this code, I'll log something to replace it. I just didn't want to introduce it.